python/syntax.lark

```diff
@@ -36,13 +36,17 @@ arguments: "[" (name_type (" " name_type)*)? "]"
 anonymous_func: "(" arguments "->" NAME ":" (instruction (";" instruction)*) ")"
 char: "\\" ( escape_code | "{" /./ "}")
 escape_code: /[tnr]/
+tupledef: "(" NAME ("," NAME)* ")"
+tupleval: "(" value ("," value)* ")"
```

Comment:
I think this would allow a tuple of a single item, which might cause a syntactical ambiguity. Should we have a distinction between normal values and tuples of one item?

Reply:
I am not sure, and cannot tell why one length tuples would be bad?

Reply:
It just seems to be confusing to have a distinction between tuples of one item and non-tuples